src/main/java/com/aspire/student/util/StudentUtil.java

```diff
@@ -26,6 +26,7 @@
    */
   public static List<StudentVO> getStudentVOList(List<Student> studentList) {
     List<StudentVO> studentVOList = new ArrayList<StudentVO>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2293)